src/sentry/models/sentryapp.py

```diff
@@ -54,7 +54,7 @@
 
 
 def default_uuid():
-    return six.binary_type(uuid.uuid4())
+    return six.binary_type(six.text_type(uuid.uuid4()).encode("ascii"))
```

Comment:
Since this is being used as a default for a charfield is the correct thing to just convert it to a `text_type`?

Reply:
That seems correct, I didn't question it at the time. Seems to have been introduced in https://github.com/getsentry/sentry/pull/8424/files#diff-7c7048cfe1e12f7101c1bbae1ad3ffc2R40, unsure how that got past review.